impeller/renderer/backend/gles/blit_command_gles.cc

```diff
@@ -276,7 +276,7 @@ bool BlitCopyBufferToTextureCommandGLES::Encode(
                   0u,                          // border
                   data.external_format,        // external format
                   data.type,                   // type
-                  tex_data                     // data
+                  nullptr                      // data
```

Comment:
This is correctly initialized below, but needs to be nullptr, or the driver may read out of bounds if we're only updating a subregion.